synapse/rest/client/versions.py

```diff
@@ -48,7 +48,11 @@ def on_GET(self, request):
                     "r0.5.0",
                 ],
                 # as per MSC1497:
```

Comment:
this comment should be next to the thing it applies to

Reply:
[MSC1497](https://github.com/matrix-org/matrix-doc/issues/1497) is unstable features no?

Reply:
oh, my bad, sorry